css/_horizontal-navigation.scss

```diff
@@ -13,25 +13,25 @@
     border-bottom: solid 1px var(--color-neutral-100);
     list-style: none;
     margin: 0 calc(-1 * var(--space-medium));
-    
-    @media screen and ($breakpoint-large) {
+
+    @media screen and ($breakpoint-medium) {
```

Comment:
The reason why the breakpoint was converted to `large` was because of how long the horizontal navigation is in [Past Activity](https://account.lib.umich.edu/past-activity/u-m-library). The list will start stacking on top of each other before converting to the "mobile" design. If you really want to stick with `medium`, then I would maybe reduce the left padding between each list item when the `font-size` decreases.